metricbeat/module/etcd/self/_meta/data.json

```diff
@@ -9,8 +9,8 @@
             "id": "8e9e05c52164694d",
             "leaderinfo": {
                 "leader": "8e9e05c52164694d",
-                "starttime": "2017-12-07T07:20:33.241031712Z",
-                "uptime": "9m4.919923089s"
```

Comment:
For another PR, we could convert this to seconds.

Reply:
Can you open an issue to enhance this, please?